### PR TITLE
latest Zanata translation changes in kie-wb-common from 2016-11-15

### PR DESCRIPTION
**Pull request by kiereleaseuser**

### CODE REVIEW

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/resources/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants_de.properties [hunk 1]
```diff
@@ -156,7 +156,7 @@ Saving=Speichern …
 # translation auto-copied from project KIE Workbench - Common, version 6.0.0, document org.kie.workbench.widgets/kie-wb-common-ui/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants, author jdimanos
 UploadSuccess=Erfolgreich hochgeladen
 # translation auto-copied from project KIE Workbench - Common, version 6.0.0, document org.kie.workbench.widgets/kie-wb-common-ui/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants, author jdimanos
-InvalidFileName0=Ressourcenname "{0}" ist ungültig.
+InvalidFileName0=Ressourcenname »{0}« ist ungültig.
```
Comment (manstis) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/resources/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants_de.properties]: @hasys IMO if the person who performed the German translations decided to replace double-quotes with the shown "special character" then it's not for me; or us to question. It should not cause any problems for compilation.

Comment (manstis) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/resources/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants_de.properties]: There are many, many places this replacement has been performed (almost like the person who performed the German translations deliberately went through potentially **all** i18n files to make the change).

Comment (hasys) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/resources/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants_de.properties]: @manstis yes, but there are reversed aren't they? I will try to check it.

Comment (hasys) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/resources/org/kie/workbench/common/widgets/client/resources/i18n/CommonConstants_de.properties]: @manstis Ok, I checked it, it looks correct :) Thank you!

**Pull request merged**